apps/public-docsite-v9/src/Theme/ThemeDesigner.css

```diff
@@ -0,0 +1,3 @@
+#docs-root .sbdocs-content {
```

Comment:
why is this needed ? "overriding" storybook magic globals is not very good idea as things might break (especially if we dont have any test coverage )

Reply:
My goal is to hide the theme selector and allow the story to be full width....otherwise you get this:

![image](https://user-images.githubusercontent.com/1434956/187239642-e1656174-45be-4130-ada1-3e2e81a973fa.png)

I'll work on a less hacky way to do it.

Reply:
Ok, I moved both the theme removal and the full screen functionality to reusable parameters.